src/wowman/curseforge_api.clj

```diff
@@ -41,25 +42,72 @@
         stable-releases (filterv #(= (:releaseType %) stable) latest-files)
 
         ;; no alternative versions, for now
-        stable-releases (remove :exposeAsAlternative stable-releases)
+        stable-releases (remove :exposeAsAlternative stable-releases)]
 
-        ;; I don't know if it's possible, but a group may still have more than one result
-        ;; results are ordered and group-by preserves ordering, so take the first
-        ]
+    ;; I don't know if it's possible, but a group may still have more than one result
+    ;; results are ordered and group-by preserves ordering, so take the first
     (group-by :gameVersionFlavor stable-releases)))
 
+(defn latest-versions-by-gameVersion
+  "given a curseforge-api result, returns a map of release data
+  uses :gameVersion which, if present, indicates the game tracks a release supports
+  prefer this over `latest-versions-by-game-track`"
+  [api-result]
+
+  ;; issue#63: curseforge actually allow a release to be on both retail and classic game tracks
+  ;; the single value under "gameVersionFlavor" is inaccurate and misleading then and we can't trust it
+  ;; instead, we must look at the list "gameVersion" and convert the numbers we find there into game tracks
```

Comment:
"instead we look at the "gameVersion" list and convert the versions we find there into game tracks."